### PR TITLE
alert: verify test for distance/within keywords

### PR DESCRIPTION
**Pull request by jmtaylor90**
* this is based on https://redmine.openinfosecfoundation.org/issues/1

Signed-off-by: jason taylor <jtfas90@gmail.com>

### CODE REVIEW

File: tests/alert-distance-within-1/suricata.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+%YAML 1.1
+---
+
+include: ../../etc/suricata-4.0.3.yaml
```
Comment (victorjulien) [inline tests/alert-distance-within-1/suricata.yaml]: this shouldn't be necessary

File: tests/alert-distance-within-1/test.yaml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+requires:
+  features:
+    - HAVE_LIBJANSSON
+
+checks:
+
+  # Check that we only have one alert event type in eve.
```
Comment (victorjulien) [inline tests/alert-distance-within-1/test.yaml]: this checks that there are 2 alert records, but doesn't say anything about other record types

Comment (jmtaylor90) [reply tests/alert-distance-within-1/test.yaml]: Ah, good point, I will update. Thanks!

File: tests/alert-distance-within-1/suricata.yaml (hunk 1 above)
Comment (jmtaylor90) [reply tests/alert-distance-within-1/suricata.yaml]: will fix

**Comment by jmtaylor90** [comment]
followed up in #36

**Pull request closed without merging**